wp-content/plugins/jetpack/_inc/build/crowdsignal-shortcode.min.js

```diff
@@ -0,0 +1 @@
+!function(){var e;e=document,"undefined"!=typeof jQuery&&jQuery(e.body).trigger("pd-script-load")}();
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).